assets/helpers/contributions.js

```diff
@@ -94,6 +99,22 @@ function parse(input: ?string, contrib: Contrib): ParsedContrib {
 
 }
 
+function betterParse(input: string, contributionType: Contrib): ParsedAmount {
```

Comment:
Is there a more meaningful name than `betterParse`? What are we parsing here?

Reply:
If it is an improvement over the parse method, can we replace that method?

Reply:
"The old one now just exists for the old contributions component, and will go away when the non-circles pages do."

Reply:
Fair enough, Is it possible to come up with a  better name?